### PR TITLE
add Tom's Fairport file to master

### LINKED ISSUE #4: Create a file for ThomasLinder Fairport, NY

### PR DESCRIPTION
**Pull request by thomaslinder**
closes #4

### CODE REVIEW

File: ThomasLinder-Fairport.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Fairport, NY
+
+#### Places to Go
```
Comment (khusban) [inline ThomasLinder-Fairport.md:3]: some comment

**Comment by thomaslinder** [comment]
@chrisatkins Does my pull request look ok to :ship: ?
:+1:

**Comment by thomaslinder** [comment]
:8ball:

**Comment by chrisatkins** [comment]
> Does my pull request look ok to  ?
> You betcha

**Pull request merged**